tests/foreman/cli/test_oscap.py

```diff
@@ -57,16 +57,26 @@ def test_positive_list_default_content_with_admin(self, module_target_sat):
             1. Login to shell from admin account.
             2. Execute the scap-content command with list as sub-command.
 
-        :expectedresults: Default scap-content are listed.
+        :expectedresults: Default scap-content are listed. They should already be populated on a fresh Satellite instance from Broker.
 
         :BZ: 1749692
+        :verifies: SAT-30938
 
         :customerscenario: true
 
         :CaseImportance: Medium
         """
-        scap_contents = [content['title'] for content in module_target_sat.cli.Scapcontent.list()]
-        assert f'Red Hat rhel{module_target_sat.os_version.major} default content' in scap_contents
+        scap_contents = [content['title'] for content in satellite_host.cli.Scapcontent.list()]
+        for rhel_version in range(6, 11):
+            assert f'Red Hat rhel{rhel_version} default content' in scap_contents
+        res = satellite_host.execute("rpm -q scap-security-guide-satellite")
+        assert res.status == 0
+        res = satellite_host.execute("rpm -ql scap-security-guide-satellite")
+        for rhel_version in range(6, 11):
```

Comment:
I wonder if we should rather rely on the supportability matrix like here https://github.com/SatelliteQE/robottelo/blob/9679c34794bdef42734d61d90cb525f0eba27feb/tests/foreman/ui/test_contenthost.py#L94 maybe even re-use that fixture. It could be more future-proof, on the other hand we no longer have rhel6 there

Reply:
You're right, including the OTOH part so I wouldn't do that.